### PR TITLE
ORC-546 - Allow visitors to create projects associated with practices

### PR DESCRIPTION
**Pull request by tiagojsag**

### CODE REVIEW

File: client/src/app/(modules)/practices/(main)/(details)/parsers.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ export const getPracticeFields = (practice: Practice): FieldType[] => {
 
   if (projects && projects?.data?.length) {
```
Comment (clementprdhomme) [inline client/src/app/(modules)/practices/(main)/(details)/parsers.ts:50]: You need to do further changes here. The acceptance criteria says:

- The field is named “Initiative”, dynamically pluralised
- The field is located after: “Organisations” (currently “Institutions”)
- **The field’s value is either a list of links to the related projects (if it exists) or else the value of the “Project Fund” field in Strapi**

The last point means that if `projects.length` is `0`, then we display `projectName` instead (the content of the previous condition). The field “Initiative Name” would disappear.

File: client/src/app/(modules)/practices/(main)/(details)/parsers.ts [hunk 2]
```diff
@@ -49,7 +49,7 @@ export const getPracticeFields = (practice: Practice): FieldType[] => {
...
     fields.push({
-      label: 'Project',
+      label: `Project${projects?.data?.length > 1 ? 's' : ''}`,
```
[the remark this answers is not in the document]
Comment (tiagojsag) [reply client/src/app/(modules)/practices/(main)/(details)/parsers.ts:52]: right, still haven't fully made the metal switch to the new name. Will make the change

File: client/src/app/(modules)/practices/(main)/(details)/parsers.ts (hunk 1 above)
Comment (tiagojsag) [reply client/src/app/(modules)/practices/(main)/(details)/parsers.ts:46]: sorry, missed that. should be fixed now

File: client/src/app/(modules)/practices/(main)/(details)/parsers.ts [hunk 3]
```diff
@@ -43,16 +43,14 @@ export const getPracticeFields = (practice: Practice): FieldType[] => {
...
-
   if (projects && projects?.data?.length) {
     fields.push({
-      label: 'Project',
+      label: `Initiative${projects?.data?.length > 1 ? 's' : ''}`,
       value: projects?.data?.map((project) => project.attributes?.name),
       url: projects?.data?.map((project) => `/network/initiative/${project.id}`),
     });
+  } else if (projectName) {
+    fields.push({ label: 'Initiative Name', value: projectName });
```
Comment (clementprdhomme) [inline client/src/app/(modules)/practices/(main)/(details)/parsers.ts:53]: Can you name this “Initiative” as well?

**Pull request merged**